benchmarks_test.go

```diff
@@ -215,9 +221,11 @@ func BenchmarkManyWritersDiode(b *testing.B) {
 	wg.Wait()
 	b.ResetTimer()
 	b.RunParallel(func(pb *testing.PB) {
+		var i int
```

Comment:
Why not use the for loop `for i := 0; pb.Next(); i++ { ... }`

Reply:
I think that reads weird. At first glance it would look like `i` is used somehow to exit the loop.

Reply:
cool